### ISSUE TITLE
Dragonfire 1.0.1 crashes as version requires (1.0.0) is not updated

### ISSUE THREAD

**Issue by kkpal09** [DragonComputer/Dragonfire#108]
![dragonfire 1 0 1_crash](https://user-images.githubusercontent.com/29928277/53782537-ab0c9800-3edb-11e9-8260-cac872001c2a.png)

**Comment by mertyildiran** [comment]
@kkpal09 I don't understand how did happened... Did you install it via `sudo dpkg -i dragonfire_1.0.1_amd64.deb`? Cloud you share the output of `dragonfire` executed from terminal?

**Comment by lebedov** [comment]
I encountered the same issue after installing `dragonfire_1.0.1_amd64.deb` on Ubuntu 18.04; here is the stack trace:

```
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/pkg_resources/__init__.py", line 583, in _build_master
    ws.require(__requires__)
  File "/usr/local/lib/python3.6/dist-packages/pkg_resources/__init__.py", line 900, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/usr/local/lib/python3.6/dist-packages/pkg_resources/__init__.py", line 791, in resolve
    raise VersionConflict(dist, req).with_context(dependent_req)
pkg_resources.VersionConflict: (dragonfire 1.0.1 (/usr/lib/python3/dist-packages), Requirement.parse('dragonfire==1.0.0'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/bin/dragonfire", line 6, in <module>
    from pkg_resources import load_entry_point
  File "/usr/local/lib/python3.6/dist-packages/pkg_resources/__init__.py", line 3191, in <module>
    @_call_aside
  File "/usr/local/lib/python3.6/dist-packages/pkg_resources/__init__.py", line 3175, in _call_aside
    f(*args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/pkg_resources/__init__.py", line 3204, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/usr/local/lib/python3.6/dist-packages/pkg_resources/__init__.py", line 585, in _build_master
    return cls._build_from_requirements(__requires__)
  File "/usr/local/lib/python3.6/dist-packages/pkg_resources/__init__.py", line 598, in _build_from_requirements
    dists = ws.resolve(reqs, Environment())
  File "/usr/local/lib/python3.6/dist-packages/pkg_resources/__init__.py", line 786, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'dragonfire==1.0.0' distribution was not found and is required by the application
```

**Comment by mertyildiran** [comment]
@lebedov oops, my bad that I forgot to bump some of the version strings in 1.0.1 release. :sweat_smile: 

But it's fixed now with https://github.com/DragonComputer/Dragonfire/commit/ce9d76eea203630baaa4ac29ef13beaccd69f122

I have also published a [new release (1.0.2)](https://github.com/DragonComputer/Dragonfire/releases/tag/v1.0.2) so please try to install [dragonfire_1.0.2_amd64.deb
](https://github.com/DragonComputer/Dragonfire/releases/download/v1.0.2/dragonfire_1.0.2_amd64.deb)

**Comment by kkpal09** [comment]
> @kkpal09 I don't understand how did happened... Did you install it via `sudo dpkg -i dragonfire_1.0.1_amd64.deb`? Cloud you share the output of `dragonfire` executed from terminal?

I had installed using both `sudo dpkg -i dragonfire_1.0.1_amd64.deb` and `dragonfire` from terminal. I got the same backtrace as shared by @lebedov 

I had corrected the version to 1.0.1 in local copy and was able to run dragonfire. I will checkout the new release(1.0.2). Thanks for quick fix!

**Comment by lebedov** [comment]
Unfortunately, the newest package also seems to have problems; several required packages [are not installed](https://github.com/DragonComputer/Dragonfire/issues/110), and there seems to be an nltk version mismatch.
